hw4/main.js

```diff
@@ -0,0 +1,414 @@
+// ----==== JS School - Lecture 4 HW ====---- 
+
+/**
+ * Сriteria for assessment
+ * 
+ * 5 - All tasks are correctly solved (23 items), the code is clean, the solutions are optimal;
+ * 4 - Correctly solved all the tasks of the base block (15 tasks), the code is clean;
+ * 3 - At least 10 problems are correctly solved, the code is clean;
+ * 2 - Correctly solved at least 10 problems;
+ * 1 - At least 5 problems solved correctly.
+ */
+
+/**
+ * Warning
+ * 
+ * Do not rename function names or change arguments.
+ */
+
+// ----==== Basic exercises (15 items) ====---- 
+/**
+  * Exercise 1
+  *
+  * Write a function that returns odd array values.
+  * [1,2,3,4] => [1,3]
+  */
+const getOddValues = numbers => {
```

Comment:
It could be written as 
`const getOddValues = numbers => numbers.filter(el => el % 2);`